### PR TITLE
ADEN-13062 Adding interstitial to all N&R platforms

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/core/services/slot-service.ts [hunk 1]
```diff
@@ -254,7 +254,7 @@ class SlotService {
 	 */
 	private getGptAdSlot(id: string): AdSlot {
 		const prefix = context.get('custom.serverPrefix');
```
Comment (rafalgiemza) [inline src/core/services/slot-service.ts:256]: Maybe put empty string as a fallback here? 

```suggestion
		const prefix = context.get('custom.serverPrefix') ?? '';
```

File: src/platforms/news-and-ratings/metacritic/setup/context/slots/metacritic-slots-context.setup.ts [hunk 2]
```diff
@@ -5,6 +5,19 @@ import { Injectable } from '@wikia/dependency-injection';
 export class MetacriticSlotsContextSetup implements DiProcess {
 	execute(): void {
 		const slots = {
+			interstitial: {
```
Comment (rafalgiemza) [inline src/platforms/news-and-ratings/metacritic/setup/context/slots/metacritic-slots-context.setup.ts:8]: This part repeats for all N&R platform. Maybe we can put it in shared folder?

**Review by rafalgiemza — APPROVED** [review]
Looks good to me, I have only 2 suggestions

File: src/core/services/slot-service.ts (hunk 1 above)
Comment (dmnsobczak) [reply src/core/services/slot-service.ts:256]: No, because we want to skip the dot `.` too when server prefix is not set.

**Review by kwfandom — APPROVED** [review]

**Pull request merged**